docs/src/main/asciidoc/kubernetes-resources.adoc

```diff
@@ -392,4 +393,68 @@ Below you will find tables describing all available types.
 | registry | String |   |  |
 | auto-push-enabled | boolean |   | false 
 | auto-build-enabled | boolean |   | false 
+|====
+
+=== Openshift support
+
+To enable the genration of Openshift resources, you need to include Openshift in the target platforms:
+
+[source]
+----
+kubernetes.deployment.target=openshift
+----
+
+If you need to genrate resources for both platforms (vanilla kubernetes and opesnhift), then you need to include both.
+
+[source]
+----
+kubernetes.deployment.target=kubernetes openshift
```

Comment:
Is this comma whitespace separated or comma separated?

Reply:
whitespace

Reply:
We use comma's everywhere else, it should also use comma's for consistency.

Is there any reason why this is not using the Quarkus configuration model, but is doing its own config?

Reply:
Switched to `comma separated`.

Regarding the config, this pull request does get the config from the Quarkus `ConfigProvider` and converts it to the `dekorate` format. Does this anser your question @stuartwdouglas or you meant something else?

Reply:
@stuartwdouglas The configuration of this extension uses the pass-through strategy like the Kafka Streams does. 
Originally this extension had the regular Quarkus configuration but it was too limiting so it was switched as part of #4308 (see [this](https://github.com/quarkusio/quarkus/pull/4308#discussion_r330445280) for more details)